ext/js/comm/anki-connect.js

```diff
@@ -708,6 +708,7 @@ export class AnkiConnect {
                 noteId: note,
                 cardId,
                 flags: typeof flags === 'number' ? flags : 0,
+                queue: typeof queue === 'number' ? queue : 0,
```

Comment:
Even though it's what Anki calls it, `queue` is a terrible name for this. Maybe `cardState`.